controllers/userController.js

```diff
@@ -0,0 +1,35 @@
+const userModel = require("../models/userModel");
+const uuid = require("uuid");
+const SHA256 = require("crypto-js/sha256");
+const jwt = require("jsonwebtoken");
```

Comment:
there should be a new package.json included in this commit to reflect these new packages added